packages/now-cli/src/util/prompt-options.js

```diff
@@ -12,12 +12,14 @@ function promptOptions(opts) {
       const s = v.toString();
 
       const cleanup = () => {
-        if (process.stdin.setRawMode) {
-          process.stdin.setRawMode(false);
-        }
+        if (process.stdin) {
```

Comment:
It's weird to me because I can't think of a scenario where `process.stdin` is actually `undefined`. AFAIK it's always a readable stream 🤔

Reply:
I'm not sure about this one either, I've just added the check to all placers I found to be sure.